pwiz/data/msdata/SpectrumList_MGF.cpp

```diff
@@ -235,6 +235,16 @@ class SpectrumList_MGFImpl : public SpectrumList_MGF
                                 else
                                     selectedIon.set(MS_selected_ion_m_z, value, MS_m_z);
                             }
+                            else if (name == "INVREION")
```

Comment:
This is what timsconvert does? I would think these should be in the TITLE string but if timsconvert (or some Matrix Science product like Distiller) does it I suppose that's fine.

Reply:
I´m not sure, I will check this.